redis/src/main/scala/zio/redis/api/Hashes.scala

```diff
@@ -240,6 +240,41 @@ trait Hashes {
     val command = RedisCommand(HVals, ArbitraryInput[K](), ChunkOutput(ArbitraryOutput[V]()))
     command.run(key)
   }
+
+  /**
+   * Returns a random field from the hash value stored at `key`.
+   *
+   * @param key of the hash which fields should be read
+   * @return random field in the hash or None when `key` does not exist.
+   */
+  final def hRandField[K: Schema, V: Schema](key: K): ZIO[RedisExecutor, RedisError, Option[V]] = {
+    val command = RedisCommand(HRandField, ArbitraryInput[K](), OptionalOutput(ArbitraryOutput[V]()))
+    command.run(key)
+  }
+
+  /**
+   * Return an array of distinct fields. The array's length is either `count` or the hash's number of fields if `count` is greater.
```

Comment:
Please keep the document consistent with other commands.

Reply:
1. @param key of a sorted set
2. @param key Key of a sorted set 

There are still two styles of documentation, but I'm not sure which is the standard(Maybe the second one? Scala source code uses the second one.). An additional PR may be needed to unify all document comments.

Reply:
Definitely the second one. However, if there are no proboems with the implementation, let's merge this and do the docs unification in the follow up.